src/evict/evict_lru.c

```diff
@@ -188,6 +189,29 @@ __wt_evict_list_clear_page(WT_SESSION_IMPL *session, WT_REF *ref)
         __wt_spin_unlock(session, &cache->evict_queues[q].evict_lock);
     }
     WT_ASSERT(session, !F_ISSET_ATOMIC_16(ref->page, WT_PAGE_EVICT_LRU));
+}
+
+/*
+ * __wt_evict_list_clear_page --
+ *     Make sure a page is not in the LRU eviction list. This called from the page eviction code to
```

Comment:
Typo: This *is* called

Reply:
Updated in [c5a1b2d](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/c5a1b2d42338e72678268bab232723323345e9a5)